README.md

```diff
@@ -38,17 +38,20 @@ Table of Contents
 #### Running as nix flake
```

Comment:
In Ghaf documentation we use title case. 
https://github.com/tiiuae/ghaf/blob/main/docs/style_guide.md#headings

Reply:
Fixed in https://github.com/tiiuae/sbomnix/commit/b424d9b917e0de3151289f417ef927f3726e339a